test/specs/WebexMeeting/WebexMeeting.e2e.js

```diff
@@ -1,10 +1,46 @@
-describe('my widget', () => {
-  it('should find sections and match text', () => {
-    browser.url('https://webex.github.io/widgets/');
+import {createUser, removeUser} from '../../util';
+import MeetingPage from './MeetingPage';
+
+describe('Meeting Widget', () => {
+  let user;
+  let room;
+  let accessToken;
+  let meetingDestination;
+
+  beforeAll(async () => {
+    try {
+      user = await createUser();
+      room = await user.sdk.rooms.create({title: 'Test Room'});
+    } catch (e) {
```

Comment:
Don't use single character variables
```suggestion
    } catch (error) {
```